librz/include/rz_core.h

```diff
@@ -1053,7 +1053,11 @@ RZ_API bool rz_core_meta_string_add(RzCore *core, ut64 addr, ut64 size, RzStrEnc
 RZ_API bool rz_core_meta_pascal_string_add(RzCore *core, ut64 addr, RzStrEnc encoding, RZ_NULLABLE const char *name);
 
 // cprint.c
-RZ_API char *rz_core_print_string_c_cpp(RzCore *core);
+RZ_API RZ_OWN char *rz_core_print_string_c_cpp(RzCore *core);
+RZ_API RZ_OWN char *rz_core_hex_of_assembly(RzCore *core, const char *assembly);
+RZ_API RZ_OWN char *rz_core_esil_of_assembly(RzCore *core, const char *assembly);
+RZ_API RZ_OWN char *rz_core_assembly_of_hex(RzCore *core, const char *hexstr);
+RZ_API RZ_OWN char *rz_core_esil_of_hex(RzCore *core, const char *hexstr);
```

Comment:
I think these two functions should accept a `ut8 *hex, int len` instead of a string. The conversion string -> hexstr should be done outside these functions IMHO.